OpacPica-Plugin/src/main/java/org/kitodo/production/plugin/opac/pica/GetOpac.java

```diff
@@ -16,16 +16,17 @@
 import java.io.StringReader;
 import java.util.Objects;
 import java.util.concurrent.TimeUnit;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
 
+import javax.xml.XMLConstants;
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 import javax.xml.parsers.SAXParserFactory;
 
 import org.apache.commons.httpclient.HttpClient;
 import org.apache.commons.httpclient.methods.GetMethod;
-import org.apache.logging.log4j.LogManager;
```

Comment:
Why did you switch from log4j version 2 logging instance to an other? This change breaks the compilation of your pull request.

Reply:
Oh, that's why! The import organizes Eclipse on its own, and somehow it has a habit of changing that occasionally. Why, I have not found out yet.

Reply:
but reorganizing the imports shouldn't change the source, right?
If really eclipse is doing this, you should absolutely find the setting and disable it, beacuse this is not acceptable! Furthermore you should check your Pull Requests to identify such behaviour.